### DISCUSSION TITLE
Backdrop close on right mouse click

### DISCUSSION THREAD

**Discussion by dcheglakov** [melt-ui/melt-ui#107] [Q&A] [answered]
Currently, if I right-click on the backdrop of the dialog, it closes. But it seems to me that this is not quite the right behavior. 

I can't open a context menu to print a dialog or to inspect it in dev tools. 
In [Radix UI](https://www.radix-ui.com/docs/primitives/components/dialog) and [a11y-dialog](https://a11y-dialog.netlify.app/) the dialog does not close if you press the right mouse button.

Maybe Melt UI dialog should behave the same way?

**Comment by TGlide** [comment, marked as answer]
100%! This should be considered a bug.

**Reply by dcheglakov** [reply]
I'll create an issue. Thanks.
